security/integration/main.go

```diff
@@ -31,6 +34,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/uuid"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/clientcmd"
+	_ "k8s.io/client-go/plugin/pkg/client/auth" // to avoid 'No Auth Provider found for name "gcp"'
```

Comment:
What does this leading '_' mean?

Reply:
_ means imported but not used.

importing this modules load "gcp" plugin.

Without this import, integration test binary will be failed with "No Auth Provider found for name "gcp"" error message

Reply:
@wattli Please see this:
https://stackoverflow.com/questions/21220077/what-does-an-underscore-in-front-of-an-import-statement-mean-in-golang/21225073#21225073
I also just learned it :)

Reply:
Thanks for the explanation.